### PR TITLE
Request form styling

### PR DESCRIPTION
**Pull request by dwheal01**
8B11249B-A6CA-4016-A1B8-82D54576400D

### CODE REVIEW

File: src/components/Form.jsx [hunk 1]
```diff
@@ -74,4 +77,189 @@ export const AcceptanceForm = ({request, handleClose}) => {
...
+
+            {/* <Row className="mt-3 justify-content-center">
+                <Col xs="auto">
+                    <p>
+                        Selected Time: Day {selectedDay || 'No day'} at{' '}
+                        {selectedHour !== '' ? `${selectedHour} hour(s)` : 'No hour'} and{' '}
+                        {selectedMinute !== '' ? `${selectedMinute} minute(s)` : 'No minute'}
+                    </p>
+                </Col>
+            </Row> */}
```
Comment (Herbert540) [inline src/components/Form.jsx]: Can you remove unused commented out code

File: src/components/Form.jsx [hunk 2]
```diff
@@ -74,4 +77,189 @@ export const AcceptanceForm = ({request, handleClose}) => {
...
+                        onSelect={(e) => {setSelectedMinute(e); setTimer(selectedDay*24*60+selectedHour*60+e*1)}}
+                    >
+                        <div style={dropdownMenuStyle}>
+                            {minutes.map((minute) => (
+                                <Dropdown.Item key={minute} eventKey={minute}>
+                                    {minute < 10 ? `0${minute}` : minute} Minute(s)
+                                </Dropdown.Item>
+                            ))}
+                        </div>
+                    </DropdownButton>
```
Comment (Herbert540) [inline src/components/Form.jsx:148]: Nit: Looks like this is a lot of repeated code, could you make it reusable maybe

File: src/App.css [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
   flex-direction: column;
-  min-height: 100vh;
+  /* min-height: 100vh; */
 }
 
-.content {
+/* .content {
   padding-bottom: 50px;
-}
+} */
```
Comment (Herbert540) [inline src/App.css:9]: Can you remove unused comment

File: src/components/Form.jsx [hunk 4]
```diff
@@ -74,4 +77,189 @@ export const AcceptanceForm = ({request, handleClose}) => {
...
 
+const TimeSelector = ({setTimer}) => {
+    const [selectedDay, setSelectedDay] = useState('');
+    const [selectedHour, setSelectedHour] = useState('');
+    const [selectedMinute, setSelectedMinute] = useState('');
+
+    const days = Array.from({ length: 8 }, (_, i) => i); // Generates array [1, 2, 3, ..., 7]
+    const hours = Array.from({ length: 24 }, (_, i) => i);  // Generates array [0, 1, 2, ..., 23]
+    const minutes = Array.from({ length: 60 }, (_, i) => i);  // Generates array [0, 1, 2, ..., 59]
+
```
Comment (Herbert540) [inline src/components/Form.jsx:88]: I'm confused about the pre-defined `days`, `hours`, and `minutes`. Are we restricting the user  to only these values? I may have missed it

**Review by Herbert540 — APPROVED** [review]
lgtm!

**Comment by NoodleSoup0** [comment]
Looks same as screenshot

**Review by NoodleSoup0 — APPROVED** [review]
Everything looks good! Function as expected from ss

**Pull request merged**